File: tecjobs-api/src/main/java/com/raffdevs/tecjobsapi/entities/Company.java
package com.raffdevs.tecjobsapi.entities;

import com.fasterxml.jackson.annotation.JsonIgnore;

import javax.persistence.*;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

@Entity
@Table(name = "company")
public class Company implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(nullable = false)

    private String name;
    @Column(name = "about_us", nullable = false)
    private String aboutUs;

    @Column
    @OneToMany(mappedBy = "company")
    @JsonIgnore
    private List<Employment> employments = new ArrayList<>();

    public Company() {}

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAbout_us() {
        return aboutUs;
    }

    public void setAboutUs(String aboutUs) {
        this.aboutUs = aboutUs;
    }

    public List<Employment> getEmployments() {
        return employments;
    }

    public void setEmployments(List<Employment> employments) {
        this.employments = employments;
    }
}

File: tecjobs-api/techjobs-api-legacy/techjobs-api/Src/Entities/Company.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace techjobs_api.Src.Entities
{
    [Table("company")]
    public class Company
    {
        [Key]
        public int Id { get; set; }

        [Display(Name = "name")]
        public string Name { get; set; }

        [Display(Name = "employments")]
        public List<Employment> Employments {  get; set; } 
    }
}
